controller/ApplicationsController.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require("mongoose");
+const CustomError = require("../utils/customError");
+const responseHandler = require("../utils/responseHandler");
+// Application model
+const Applications = require("../models/applications");
+const Organizations = require("../models/organizations");
+
+exports.getAllApplications = async (req, res, next) => {
+  try {
+    const { orgId } = req.params;
+    if (!mongoose.Types.ObjectId.isValid(orgId)) {
+      return next(new CustomError(400, "Invalid ID"));
+    }
+
+    const organization = await Organizations.find({ _id: orgId });
+    if (!organization) {
+      return next(new CustomError(400, "organizations not found"));
+    }
+
+    const applicationId = await Applications.findById({});
```

Comment:
This should be a `find` not `findById` as you are returning all applications of an organization not just one application. iAlso include the `organizationId` in your `find` query to limit the search to only applications belonging to the organization. Check previous comment on how `organizationId` is gotten